TabloidCLI/Repositories/BlogRepository.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Data.SqlClient;
+using TabloidCLI.Models;
+using TabloidCLI.Repositories;
+
+namespace TabloidCLI.Repositories
+{
+    internal class BlogRepository : DatabaseConnector,
+        IRepository<Blog>
+    {
+        public BlogRepository(string connectionString) : base
+            (connectionString) { }
+
+        public void Delete(int id)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Blog Get(int id)
+        {
+            throw new NotImplementedException();
+        }
+
+        public List<Blog> GetAll()
+       {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+              using (SqlCommand cmd = conn.CreateCommand())
+               {
+                  cmd.CommandText = @"";
+                }
+                return GetAll();
```

Comment:
pretty sure this will cause a stack overflow if you called it...

Reply:
replace the code in the method with `throw new NotImplementedException();` like what you have in the rest of the methods.